portfolio/src/main/webapp/index.html

```diff
@@ -3,11 +3,12 @@
   <head>
     <meta charset="TDF-8">
     <title>My Portfolio</title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyBstuIEgosdt30hw52bAw7vG1AqHXeWvfo"></script>
     <link href="https://fonts.googleapis.com/css2?family=Caveat&family=Montserrat&family=Nunito+Sans&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body onload="getMessage()" class="home">
+  <body onload="getMessage(); createMapSingapore(); createMapHongKong(); fetchBlobstoreUrlAndShowForm(); checkIfLoggedIn();" class="home">
```

Comment:
didn't you move this to a function in a previous commit? it should be all within a load function.

Reply:
yep just realized wasn't merging the changes to master